tests/test_analytics.py

```diff
@@ -22,11 +22,15 @@
 from reportportal_client.external.constants import GA_ENDPOINT, GA_INSTANCE
 from reportportal_client.external.google_analytics import send_event
 
+TEST_PYTHON_VERSION = '3.6.6'
```

Comment:
No need to keep this variable. Just use `return_value=3.6.6` when patching the version.